schema/va-spec/base/experimental-functional-impact-study-statement-profile-source.yaml

```diff
@@ -0,0 +1,101 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://w3id.org/ga4gh/schema/va-spec/1.x/base/experimental-functional-impact-study-statement-profile-source.yaml"  
+title: Experimental Variant Functional Impact Statement
+strict: true
+
+imports:
+  cat-vrs: ../../cat-vrs/cat-vrs-source.yaml
+  vrs: ../../vrs/vrs-source.yaml
+  gks-core: ../../gks-core/gks-core-source.yaml
+
+$defs:
+  ExperimentalVariantFunctionalImpactStudyStatement:
+    inherits: gks-core:Statement
+    maturity: draft
+    type: object
+    description: >-
+      A statement reporting a conclusion from a single assay or study about the functional
+      impact of a variant on a sequence feature (typically a gene or gene product).
+    properties:
+      type:
+        extends: type
+        const: "ExperimentalVariantFunctionalImpactStudyStatement"
+        default: "ExperimentalVariantFunctionalImpactStudyStatement"
+        description: MUST be "ExperimentalVariantFunctionalImpactStudyStatement".
+      subjectVariant:
+        extends: subject
+        oneOf:
+          - $ref: "/ga4gh/schema/vrs/2.x/json/MolecularVariation"
+          - $ref: "/ga4gh/schema/cat-vrs/1.x/json/CategoricalVariant"
```

Comment:
@ahwagner this is sort of interesting and odd and hard to explain IMO. CategoricalVariant is vast in terms of what can be represented but MolecularVariation is trying to limit it to just contextual variants on a molecule. Should we be considering whether or not we provide some way to group `systemic` vs `molecular` catVars?  If it is important at the context VRS level it seems like it would also have the same or similar importance in cat-vrs...right?

Reply:
It's an interesting thought, but not a problem we need to solve immediately. Easy to add constraints later.

However, I wonder if we would be doing ourselves a favor by leaving this a little broader right now, allow for any VRS variation, not just molecular.